source/indexes/atlas-search-index.txt

```diff
@@ -78,10 +99,17 @@ The following sections show how to create a single Atlas Search index with stati
 Create a Search Index with Static Mappings
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Use static mappings in your Atlas Search index to specify the fields you want to index and configure index options for individual fields.
+Use static mappings in your Atlas Search index to specify the fields that you want to index and configure index options for individual fields.
+
+To create a single Atlas Search index with static mappings, first create a ``definition`` document containing the ``mappings`` field.
```

Comment:
Q: Do you use static mappings to create the index, or does the index contain/use them?

Reply:
I wrote this following the style used on the [Define Field Mappings](https://www.mongodb.com/docs/atlas/atlas-search/define-field-mappings/) page in sentences like:

> Use static mappings to configure index options for fields that you don't want indexed dynamically, or to configure a single field independently from others in an index.

But this page also states that the index "uses" static / dynamic mappings:

> You can configure an entire index to use dynamic mappings, ...

Reply:
Got it. The point of this is that the [SG](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-using:~:text=Use%20by%20using%2C%20or%20recast%20the%20sentence.) discourages "using" or "with", since they can be ambiguous (as here).

I would go with "index that uses static / dynamic mappings"